.github/dependabot.yml

```
@@ -0,0 +1,28 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "04:00"
+      timezone: Europe/Warsaw
+    labels:
+      - dependencies
+    allow:
+      # Quarkus
+      - dependency-name: io.quarkus:*
+      # Zalando
+      - dependency-name: org.zalando:problem
+      # Assertj
+      - dependency-name: org.assertj:assertj-core
+      # Jmh
+      - dependency-name: org.openjdk.jmh:*
+      # jsr305
+      - dependency-name: com.google.code.findbugs:jsr305
+      # Maven plugins
```

One suggestion, some comments seems redundant, those lines are super clear, so I think comments make sense to group them together somehow. How about:
```suggestion
      # Runtime / test dependencies
      - dependency-name: io.quarkus:*
      - dependency-name: com.google.code.findbugs:jsr305
      - dependency-name: org.zalando:problem
      - dependency-name: org.assertj:assertj-core
      - dependency-name: org.openjdk.jmh:*

      # Maven plugins
```

---

Sure, I can squash these comments into one